src/Sulu/Component/Content/Compat/DataNormalizer.php

```diff
@@ -17,8 +19,10 @@ class DataNormalizer
      * @param mixed $data
      * @param mixed $state Translates to the workflow state
      */
-    public function normalize($data, $state, $parentUuid)
+    public static function normalize(FormEvent $event)
```

Comment:
The normalizer is now a Form Event listener. Note that this will be unnecessary once we change the data format sent by the admin interface (which is a MUST as all of the fields below are effectively blacklisted for use as names of conten properties).